community/bolt/src/main/java/org/neo4j/bolt/v1/messaging/msgprocess/RunCallback.java

```diff
@@ -38,6 +38,10 @@ public RunCallback( Log log )
     public void result( StatementMetadata result, Void none ) throws Exception
     {
         successMetadata.put( "fields", result.fieldNames() );
+        if ( !result.queryType().isEmpty() )
```

Comment:
Just curious, can we get a cypher query that we cannot deduce the query type after we execute/parse it?